src/js/utils.js

```diff
@@ -1,8 +1,8 @@
-export const parser = Range.prototype.createContextualFragment.bind(
-  document.createRange(),
-);
-
-export function randomID() {
-  const digits = String(Math.random()).split('.')[1].substr(0, 10);
-  return `ID${digits}`;
-}
+export const parser = Range.prototype.createContextualFragment.bind(
+  document.createRange(),
+);
+
+export function randomID() {
+  const digits = String(Math.random()).split('.')[1].substr(0, 10);
+  return `ID${digits}`;
+}

```

Comment:
- [ ] You can test this function whether it is returning the required output or not

Reply:
- [ ] This is to return a random number, therefore it's untestable.